### PR TITLE
Make dependencies consistent again

### PR DESCRIPTION
**Pull request by tiagojsag**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,8 @@
 Flask==0.12.0
-earthengine-api==0.1.102
+earthengine-api==0.1.140
```
Comment (tiagojsag) [inline requirements.txt]: this is needed by sql2gee 0.2.5

File: requirements.txt [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
 CTRegisterMicroserviceFlask==0.4.0
 pycrypto==2.6.1
 requests==2.12.4
 sql2gee==0.2.5
-oauth2client==4.1.2
+google-api-python-client==1.7.3
```
Comment (tiagojsag) [inline requirements.txt]: this is needed by something, not sure what

File: requirements.txt [hunk 3]
```diff
@@ -1,7 +1,8 @@
...
+oauth2client==4.1.3
```
Comment (tiagojsag) [inline requirements.txt]: this is missing. Somehow, docker installs this, but if you run the app natively, you'll need it.

**Comment by aagm** [comment]
Tested on my local, i haven't found any issue so it can go to develop

**Pull request merged**